### PR TITLE
SEO-178105-JS-dropped-keyword

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @gowthamanthavasiyappan ,
Please review the changes.
Thanks.
Regards,
Edith.

### CODE REVIEW

File: js/ColorPicker/Getting-Started.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
 ---
 layout: post
-title: Getting-Started
-description: getting started
+title: Getting Started with JavaScript Color Picker | Syncfusion
+description: Learn here all about getting started with Syncfusion Essential Studio JavaScriptColor Picker control, its elements, and more details.
```
Comment (gowthamanthavasiyappan) [inline js/ColorPicker/Getting-Started.md:4]: Add space between control and platform - JavaScript Color Picker

File: js/TreeView/Getting-Started.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: Getting Started with JavaScript TreeView Control | Syncfusion
-description: Learn here about getting started with Syncfusion Essential Studio JavaScript TreeView control, its elements, and more.
+description: Learn here all about getting started with Syncfusion Essential Studio JavaScript TreeView control, its elements, and more details.
```
Comment (gowthamanthavasiyappan) [inline js/TreeView/Getting-Started.md:4]: Revert this

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: js/ColorPicker/Getting-Started.md (hunk 1 above)
Comment (Edithabich) [reply js/ColorPicker/Getting-Started.md:4]: resolved

File: js/TreeView/Getting-Started.md (hunk 2 above)
Comment (Edithabich) [reply js/TreeView/Getting-Started.md:4]: reverted

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**